rosdeps.yaml

```diff
@@ -10,21 +18,72 @@ python:
       packages: [pip, setuptools, nose]
 python-imaging:
   osx:
-    homebrew:
-      packages: [homebrew/python/pillow]
+    pip:
+      packages: [pillow]
```

Comment:
This isn't actually necessary; it's present here:

https://github.com/ros/rosdistro/blob/57b647e17fe1307391a9e94e2352ee401101c059/rosdep/python.yaml#L1293-L1295